bitwarden_license/src/Portal/Controllers/SsoController.cs

```diff
@@ -41,7 +41,7 @@ public async Task<IActionResult> Index()
             }
 
             if (!_enterprisePortalCurrentContext.SelectedOrganizationDetails.UseSso ||
-                !_enterprisePortalCurrentContext.AdminForSelectedOrganization)
+                !_enterprisePortalCurrentContext.CanManagePoliciesForSelectedOrganization)
```

Comment:
Why can they edit SSo if they have policy permissions? Seems unrelated?

Reply:
My justification for this was tied to Policy people being able to enable/disable sso related polices, but there is room for distinction there. I added a `ManageSso`.